Rakefile

```diff
@@ -1,8 +1,33 @@
-require "yast/rake"
+# Copyright (c) 2018 SUSE LLC.
+#  All Rights Reserved.
+
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of version 2 or 3 of the GNU General
+#  Public License as published by the Free Software Foundation.
+
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+#  GNU General Public License for more details.
+
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, contact SUSE LLC.
+
+#  To contact SUSE about this file by physical or electronic mail,
+#  you may find current contact information at www.suse.com
+
+require 'yast/rake'
 
 Yast::Tasks.configuration do |conf|
   # The package does not live in the official YaST:Head OBS project
-  conf.obs_project = "home:thutterer"
+  conf.obs_project = 'systemsmanagement:SCC:RMT'
   # Default target for osc:build
-  conf.obs_target = "openSUSE_Factory"
+  conf.obs_target = 'openSUSE_Factory'
+  conf.skip_license_check = [ %r{^Gemfile\.lock$} ]
+end
+
+# This is required, because `yast-travis-ruby` binary calls `rake test:unit`
```

Comment:
it is defined by yast/rake so it should not be needed. see https://github.com/yast/yast-rake/blob/master/lib/tasks/test_unit.rake
but of course if you prefer you own, it is fine. You just need to clear old, otherwise it will be run twice.

Reply:
Ah, it looks for specs in 'test/', not in 'spec/' -- maybe I'll rename the 'spec/' directory.